Tests/KlaviyoCoreTests/__Snapshots__/NetworkSessionTests/testCreateEmphemeralSesionHeaders.1.txt

```diff
@@ -8,7 +8,7 @@
         - "deflate"
     ▿ (2 elements)
       - key: "User-Agent"
-      - value: "FooApp/1.2.3 (com.klaviyo.fooapp; build:1; iOS 1.1.1) klaviyo-ios/4.0.0"
```

Comment:
@ndurell for consistency with the sdk name, i did have to update the snapshot, resetting your review to make sure you're good with that. AFAIK the user agent isn't critical for anything so I think this is OK